Apps/phmfservicemanager/mfservicemanager.json

```diff
@@ -0,0 +1,2839 @@
+                    "column_order": 3
+                },
+                {
+                    "data_path": "action_result.data.*.header.InitiatedBy",
+                    "data_type": "string",
+                    "column_name": "Initiated By",
+                    "column_order": 4
+                },
+                {
+                    "data_path": "action_result.data.*.header.Open",
+                    "data_type": "boolean",
+                    "example_values": [
+                        true,
+                        false
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.header.Phase",
+                    "data_type": "string",
+                    "column_name": "Phase",
+                    "column_order": 2
+                },
+                {
+                    "data_path": "action_result.data.*.header.PlannedEnd",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.header.PlannedStart",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.header.Priority",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.header.Reason",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.header.RiskAssessment",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.header.Status",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.header.Subcategory",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.header.Title",
+                    "data_type": "string",
+                    "column_name": "Title",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "create configitem",
```

Comment:
Can this be changed to `create configuration`

Reply:
@alexa-phantom It is a community app. Do you want me to change it?

Reply:
Yes, please normalize the action name